src/main/java/com/team4/testingsystem/controllers/TestsController.java

```
@@ -161,6 +161,12 @@ public void deassignCoach(@PathVariable("testId") long testId) {
         testsService.deassignCoach(testId);
     }
 
+    @ApiOperation(value = "Use it to spend listening attempts")
+    @PostMapping(path = "/playAudio/{testId}")
+    public void spendAttempt(@PathVariable("testId") long testId) {
+        testsService.spendAttempt(testId);
```

Maybe we can throw an exception if `listeningAttempts` = 0